apps/user_status/appinfo/routes.php

```diff
@@ -34,6 +34,9 @@
 		['name' => 'UserStatus#setPredefinedMessage', 'url' => '/api/v1/user_status/message/predefined', 'verb' => 'PUT'],
 		['name' => 'UserStatus#setCustomMessage', 'url' => '/api/v1/user_status/message/custom', 'verb' => 'PUT'],
 		['name' => 'UserStatus#clearMessage', 'url' => '/api/v1/user_status/message', 'verb' => 'DELETE'],
+		// Routes for getting your backup status (if it exists)
+		['name' => 'UserStatus#getBackupStatus', 'url' => '/api/v1/user_status_backup', 'verb' => 'GET'],
```

Comment:
```suggestion
		['name' => 'UserStatus#getBackupStatus', 'url' => '/api/v1/user_status/backup', 'verb' => 'GET'],
```

To align with other endpoints?